source/manuals/request-aws-accounts.html.md.erb

```diff
@@ -1,42 +1,41 @@
 ---
-title: How to create AWS Accounts
+title: How to request Amazon Web Services (AWS) Accounts
 ---
 
 # <%= current_page.data.title %>
 
-Using multiple [Amazon Web Services (AWS)][] accounts allows GDS to:
+GDS teams can request as many AWS accounts as required, for example one production and one non-production account.
+
+Using multiple [Amazon Web Services (AWS)][] accounts allows GDS teams to:
 
 * enforce administrative isolation between workloads
 * minimize the impact of security breaches
 * isolate audit data in separate accounts
 
-GDS teams can create as many AWS accounts as they need, this means usually at least one production and one non-production account.
-
-You can read more about [AWS Multiple Account Security Strategy][] in AWS Answers.
+AWS Answers has more information about [AWS Multiple Account Security Strategy][].
 
-Requesting an account
----------------------
+## Request an account
 
 To request an AWS account complete and submit the [request an AWS account form][].
 
-Reliability Engineering will confirm if your account is eligible for inclusion in the GDS AWS bill and also perform the initial account setup.
+Reliability Engineering will confirm if your account can be included in GDS AWS billing and perform the initial account setup.
 
 Once your account is created you'll recieve an email with the account ID and will be able to assume an administrative role from [the GDS users base account][].
 
-Setting up an account
----------------------
+## Setup an account
 
 Your account will be created with a `bootstrap` role which has full admin
```

Comment:
Will people know what a bootstrap role means in this context? I understand the general concept of bootstrapping in computing, but is there a specific AWS term for the type of role we're referring to? Or, is the role people will get one that we (GDS) have already defined for them?

Reply:
The role we create is literally called `bootstrap` - this role will be there in the account to begin with so the users can assume access and set the account up. Once they've done that they should remove the `bootstrap` role.

Reply:
Great, I'll update this section to make it clear.